src/components/messenger/list/index.tsx

```diff
@@ -222,8 +228,11 @@ export class Container extends React.Component<Properties, State> {
           isMessengerFullScreen={this.props.isMessengerFullScreen}
           isFirstTimeLogin={this.props.isFirstTimeLogin}
           includeRewardsAvatar={this.props.includeRewardsAvatar}
+          userName={this.props.userName}
+          userHandle={this.props.userHandle}
```

Comment:
I'm not entirely sure this is the right field to use. It's mostly coincidence that these align with email and web3 address at the moment (particularly for new users) but a lot of existing users would have different things here and we might actually use this in the future. I think it's ok to use this right now but we should fetch the real data before calling this done.... unless the intent is to actually use the user's handle which may change to be user managed.